jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ThreadLimitHandler.java

```diff
@@ -318,13 +323,13 @@ protected void process() throws Exception
             {
                 if (LOG.isDebugEnabled())
                     LOG.debug("Threadlimited {}", _remote);
```

Comment:
Missing a space between `Thread` and `limited`. Maybe worth logging also the request and the current thread?

Reply:
I'll add the request and handler. The logger already includes the thread.